### DISCUSSION TITLE
Two JK BMS in one?

### DISCUSSION THREAD

**Discussion by dan11hh** [Louisvdw/dbus-serialbattery#140] [Q&A] [answered]
Hi and thanks a lot for this project i like a lot.
i'm just wondering if there would be a way to integrate two JK BMS, so that the victron think it is one. I'm bulding two blocks with each 16Sx280Ah LiFEPO4, with two BMSs and i don't think the victron can handle this.

**Comment by Louisvdw** [comment, marked as answer]
There will be a way yes. See  #8

**Comment by dan11hh** [comment]
Thank for your answer!
